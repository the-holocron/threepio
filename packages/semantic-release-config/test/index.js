const config = require("../releaserc.json");

describe("configuration", () => {
	describe("semantic Release", () => {
		test("should render the configuration", () => expect(config).toMatchSnapshot());
	});
});
