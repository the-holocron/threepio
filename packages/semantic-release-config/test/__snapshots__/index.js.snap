// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`configuration semantic Release should render the configuration 1`] = `
Object {
  "plugins": Array [
    Array [
      "@semantic-release/commit-analyzer",
      Object {
        "parserOpts": Object {
          "noteKeywords": Array [
            "BREAKING CHANGE",
            "BREAKING CHANGES",
            "BREAKING",
          ],
        },
        "preset": "angular",
      },
    ],
    Array [
      "@semantic-release/release-notes-generator",
      Object {
        "parserOpts": Object {
          "noteKeywords": Array [
            "BREAKING CHANGE",
            "BREAKING CHANGES",
            "BREAKING",
          ],
        },
        "preset": "angular",
        "writerOpts": Object {
          "commitsSort": Array [
            "subject",
            "scope",
          ],
        },
      },
    ],
    Array [
      "@semantic-release/changelog",
      Object {
        "changelogTitle": "# Change Log

All notable changes to this project will be documented in this file. See
[Conventional Commits](https://conventionalcommits.org) for commit guidelines.",
      },
    ],
    Array [
      "@semantic-release/npm",
      Object {
        "tarballDir": "pack",
      },
    ],
    Array [
      "@semantic-release/github",
      Object {
        "assets": "pack/*.tgz",
      },
    ],
    Array [
      "@semantic-release/git",
      Object {
        "message": "chore(release): \${nextRelease.version} [skip ci]

\${nextRelease.notes}",
      },
    ],
  ],
  "tagFormat": "\${version}",
}
`;
